.github/workflows/IntegrationTests.yml

```diff
@@ -0,0 +1,40 @@
+name: IntegrationTests
```

Comment:
How different is this workflow from that in `CI.yml`?

Reply:
Absolutely no other difference:)

I just made it separate to make it easy to see which tests are suceeding/failing.